src/core/vectortile/qgsvectortilestructure.cpp

```diff
@@ -0,0 +1,90 @@
+/***************************************************************************
+  qgsvectortilestructure.cpp
+  --------------------------------------
+  Date                 : March 2022
+  Copyright            : (C) 2022 by Nyall Dawson
+  Email                : nyall dot dawson at gmail dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgsvectortilestructure.h"
+#include "qgstiles.h"
+#include "qgsvectortileutils.h"
+#include "qgsarcgisrestutils.h"
+
+QgsVectorTileStructure QgsVectorTileStructure::fromWebMercator()
+{
+  QgsVectorTileStructure res;
+  res.setCrs( QgsCoordinateReferenceSystem( QStringLiteral( "EPSG:3857" ) ) );
+  res.setZ0xMin( -20037508.3427892 );
```

Comment:
Do we have an established convention on setZ0xMin vs setZ0xMinimum?

Reply:
You're right, fixed